packages/asset-uploader/src/schema.ts

```diff
@@ -15,6 +15,10 @@ const maxSize = z
   .transform(toBytes)
   .default(MAX_UPLOAD_SIZE);
 
+export const Env = z.object({
+  MAX_ASSETS_PER_PROJECT: z.string().default("50").transform(Number.parseFloat),
```

Comment:
probably this should be just added to both S3Env and FsEnv just like MAX_UPLOAD_SIZE?

Reply:
This Env is used in create, we can make it as a param if needed for `createAssetWithLimit`.  But not sure why